conf/charts/frontend-redis-interface/values.yaml

```diff
@@ -0,0 +1,27 @@
+app_name: "fe-rd-interface"
+
+#deployment.yaml
+replicas: 1
+image_name: "vanvalenlab/frontend-redis-interface"
+image_version: "0.1dev"
+deployment_name: "fe-rd-deployment"
+deployment_port: 8080
+container_name: "fe-rd-container"
+
+# service.yaml
+service_name: "fe-rd-service"
+service_port: 8080
+service_type: "ClusterIP"
+
+# secrets.yaml and hemlfile.d/frontend.yaml
+env:
+  REDIS_HOST: "redis-master"
+  REDIS_PORT: "6379"
+  secret:
```

Comment:
Remove leading whitespace?

Reply:
I'm a little confused. Most of these lines don't have leading whitespace. Are you talking about the variables under the `env` section.

Reply:
We will address this in a separate PR.